develop/dev-guide-use-stale-read.md

```diff
@@ -437,20 +470,24 @@ public class BookDAO {
 
 ## Session level
 
-To support reading historical version data, TiDB has introduced a new system variable `tidb_read_staleness` since version 5.4. The system variable `tidb_read_staleness` is used to set the range of historical data that the current session is allowed to read. Its data type is int and its scope is SESSION.
+To support reading historical version data, TiDB has introduced a new system variable `tidb_read_staleness` since v5.4. It is used to set the range of historical data that the current session is allowed to read. Its data type is `int` and its scope is `SESSION`.
```

Comment:
```suggestion
To support reading historical data, TiDB has introduced a new system variable `tidb_read_staleness` since v5.4. you can use it to set the range of historical data that the current session is allowed to read. Its data type is `int` and its scope is `SESSION`.
```